src/storage/store/engine.rs

```diff
@@ -45,9 +50,12 @@ pub struct ShardEngine {
     shard_store: ShardStore,
     messages_rx: mpsc::Receiver<message::Message>,
     messages_tx: mpsc::Sender<message::Message>,
+    validator_message_rx: mpsc::Receiver<snapchain::ValidatorMessage>,
+    validator_message_tx: mpsc::Sender<snapchain::ValidatorMessage>,
```

Comment:
I think we should re-use the existing channel. Change the type to be an enum that can hold a Message or a ValidatorMessage.